app/helpers/local_time_helper.rb

```diff
@@ -42,20 +48,38 @@ def utc_time(time_or_date)
   end
 
   private
+
     def find_time_format(format)
+      find_format(format, find_i18n_time_format(format), find_ruby_time_format(format), LocalTime.default_time_format)
+    end
+
+    def find_date_format(format)
+      find_format(format, find_i18n_date_format(format), find_ruby_date_format(format), LocalTime.default_date_format)
+    end
+
+    def find_format(format, i18n_format, ruby_format, default_format)
       if format.is_a?(Symbol)
-        if (i18n_format = I18n.t("time.formats.#{format}", default: [:"date.formats.#{format}", ''])).present?
+        if i18n_format
           i18n_format
-        elsif (date_format = Time::DATE_FORMATS[format] || Date::DATE_FORMATS[format])
-          date_format.is_a?(Proc) ? LocalTime.default_time_format : date_format
+        elsif ruby_format
+          ruby_format.is_a?(Proc) ? default_format : ruby_format
         else
-          LocalTime.default_time_format
+          default_format
         end
       else
-        format.presence || LocalTime.default_time_format
+        format.presence || default_format
       end
     end
 
+    def find_i18n_time_format(format); find_i18n_format(format, :time); end
```

Comment:
Let's use multi-line method definitions, since we don't do single-line anywhere else in this codebase